### PR TITLE
Adding ovn, nova and libvirt playbooks ported from dataplane services

### PR DESCRIPTION
**Pull request by jpodivin**

### CODE REVIEW

**Comment by jpodivin** [comment]
recheck

**Comment by jpodivin** [comment]
recheck

File: playbooks/ovn.yaml
Comment (fao89) [inline playbooks/ovn.yaml:1]:
```yaml
  - name: "Deploy EDPM OVN"
    hosts: "all"
    strategy: "linear"
    become: true
    tasks:
      - name: "ovn"
        import_role:
          name: "osp.edpm.edpm_ovn"
        tags:
          - "edpm_ovn"
```

Comment (jpodivin) [reply playbooks/ovn.yaml:1]: What happened here?

**Review by fao89 — APPROVED** [review]

**Review by slagle — APPROVED** [review]

**Comment by slagle** [comment]
/override rdoproject.org/github-check

rdo jobs wouldn't even test this, so overriding

**Pull request merged**